pkg/client/didexchange/client.go

```diff
@@ -45,19 +45,15 @@ func New(ctx provider) (*Client, error) {
 
 // CreateInvitation create invitation
 func (c *Client) CreateInvitation() (*InvitationRequest, error) {
-	// TODO remove nil check after provide default implementation for wallet
-	pubKey := ""
-	if c.wallet != nil {
-		keyInfo, err := c.wallet.CreateSigningKey(nil)
-		if err != nil {
-			return nil, fmt.Errorf("failed CreateSigningKey: %w", err)
-		}
-		pubKey = keyInfo.GetVerificationKey()
+	keyInfo, err := c.wallet.CreateSigningKey(nil)
```

Comment:
why a nil argument?

Reply:
nil for key metadata. for now we don't have any metadata to persist it

Reply:
I'm a bit confused: the keys sent in an invitation are for encryption purposes, not signing.

Reply:
actually, if RecipientKeys are being used to Pack() the envelope going through the wire, then these are for JWE encryption, not signing the DID/VC. 

From what it looks, this InvitationRequest being created below is to be sent over the transport. So it must be for encryption, not signing..

Reply:
@Baha-sk @llorllale https://github.com/hyperledger/aries-cloudagent-python/blob/master/aries_cloudagent/messaging/connections/manager.py#L154

Reply:
I will change it to createKey

Reply:
done